app/lib/find_constraint.rb

```diff
@@ -2,6 +2,6 @@
 
 class FindConstraint
   def matches?(request)
-    Settings.find_url&.include?(request.host) || request.host.include?('find-pr') || request.host.include?('find-review')
+    Settings.find_url&.include?(request.host) || Settings.extra_find_url&.include?(request.host) || request.host.include?('find-pr') || request.host.include?('find-review')
```

Comment:
The domain for review is https://find-review-4299.test.teacherservices.cloud/ so `request.host.include?('find-pr')` can be removed

Reply:
It also looks wrong in the test: https://github.com/DFE-Digital/publish-teacher-training/blob/52f20f4d0e29531b53754341d46be60606d529fb/spec/lib/find_constraint_spec.rb#L40

Reply:
Could the whole thing be simplified with something like `request.host.start_with? "find"` ?

Reply:
Yes, it makes sense. I was conservative in the solution because I didn't wanna break things.

But I always up for simple solutions

Reply:
Done!